Makefile

```diff
@@ -67,6 +67,9 @@ lint: bin/phpcs ## Run the linter on the PHP files
 lint-fix: bin/phpcbf ## Fix the errors detected by the linter
 	$(PHP) ./bin/phpcbf . -p -s
 
+bin/composer:
+
```

Comment:
Is this missing something?

Reply:
Yes, sorry, I committee a draft of this file by accident. I can revert or leave if to future improvements

Reply:
Probably better to revert the file or fix it up? I don't know what you were going for exactly, maybe we're only talking about a few minutes work. ^_^

Reply:
Done https://github.com/FreshRSS/FreshRSS/pull/4173/commits/e9b7ac9f10b53f295ae5a859d231cc150b7ff350